omero/sysadmins/server-upgrade.txt

```diff
@@ -59,6 +59,13 @@ your previous version of OMERO. We recommend using a future-proof symlink
 if possible, so that these stanzas do not need updating with each OMERO
 server upgrade.
 
+.. note:: Since OMERO 5.2, the OMERO web framework no longer bundles
```

Comment:
Do you want me to update across entire doc? This bit was copied over from another location.